count-note-beats.qml

```diff
@@ -304,6 +312,7 @@ MuseScore {
 		var rewindMode;
 		var toEOF;
 
+		curScore.startCmd();
```

Comment:
needed for the plugin to do anything to a score, mandatory in Mu4, optional but recommended in M3 (to have changes being added to the undo stack)

Reply:
This causes, probably harmless, warnings in 3.2.3:

```
Score::startCmd(): cmd already active
Score::endCmd(): no cmd active
```

I presume the plugin invocation is already added to the undo stack as one block (I can certainly undo them with `^Z`).

Reply:
As said, this seems needed in 4.x and at least not harm in 4.x

Reply:
I just saw that this is for dockable plugins only in 3.x, but yeah, an no harm done